schemas/com.snowplowanalytics.snowplow/web_page/jsonschema/1-1-0

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a web page context",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "web_page",
+        "format": "jsonschema",
+        "version": "1-1-0"
+    },
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "string",
+			"format": "uuid",
```

Comment:
This seems like a combination of tabs and spaces.

Reply:
Yeap fixing that.